### PR TITLE
FEATURE: migrated the chat window to svelte

### PR DESCRIPTION
**Pull request by Kharhamel**

### CODE REVIEW

File: front/src/Components/Chat/ChatElement.svelte [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    import {HtmlUtils} from "../../WebRtc/HtmlUtils";
+
+    export let message: ChatMessage
+
+    $: urlifyText = HtmlUtils.urlify(message.text);
+</script>
+
+<p>
+{#if message.type === ChatMessageTypes.text}
+    <b>{message.author}:</b> <span 	contenteditable="true" bind:innerHTML={urlifyText}></span>
```
Comment (jonnytest1) [inline front/src/Components/Chat/ChatElement.svelte:13]: wasnt there a condition or changing the current user to 'Me' ?

**Comment by jonnytest1** [comment]
so i was thinking : maybe its a better idea to use react instead of svelte - im pretty sure react has a lot more community support and for example i found something like this which should be a huge help https://www.npmjs.com/package/react-chat-elements 

syntax wise i dont think svelte and react are too different from what ive seen

**Comment by moufmouf** [comment]
@jonnytest1 We spent a huge amount of time migrating raw HTML to Svelte already.
You are right to say React has a huge ecosystem, but I'd rather avoid rewriting everything again.

+ we chose Svelte mostly because of it's store system which is really lean and does not get in the way (and plays well with Typescript too!)

File: front/src/Components/Chat/ChatElement.svelte (hunk 1 above)
Comment (Kharhamel) [reply front/src/Components/Chat/ChatElement.svelte]: You're right, I added a special case for 'me'

**Comment by jonnytest1** [comment]
@moufmouf  in the meantime i also found out that svelte is probably bigger and definitely probalby becoming bigger than i tohught so i dont think there will be a huge problem 
you might even be able to find some ready made components sometime soon

**Comment by MattGodin** [comment]
Feedbacks Chat : PR 1111

- Mettre une font plus générique
   --> Coolvetica : https://www.dafont.com/fr/coolvetica.font?text=mon+message&psize=s&back=theme
   --> Comfortaa : https://www.dafont.com/fr/comfortaa.font?l[]=10&l[]=1

- Enlever la scroll-bar ou en mettre une transparente qui s'affiche au scroll de la souris

- Pour quitter le chat, enlever la croix et mettre une "excroissance" sur le bord droit au milieu du chat

- Enlever l'heure et l'afficher uniquement lorsque le user clique sur le message (animation où le message se décale sur la droite et laisse apparaître l'heure)

- Afficher en bleu/vert un entrant et rouge un sortant.

- Fond du chat en blanc : #FAFCFC

- Bulle de texte en gris : #E7E9E9

- Bords arrondis pour la barre d'écriture (à la manière des RS d'aujourd'hui comme Messenger, Insta...)

- Bouton "Send" en rond avec image comme https://img.icons8.com/ios/452/sent.png

**Pull request merged**